### PR TITLE
Angular for gh-pages

### PR DESCRIPTION
**Pull request by eddiejaoude**
Published website available on http://werocktech.com/awesome-tech-communities/ 

screenshot 2017-06-10 08 51 04

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      true,
+      "attribute",
+      "app",
+      "camelCase"
+    ],
+    "component-selector": [
+      true,
+      "element",
+      "app",
+      "kebab-case"
```
Comment (tanyapowell) [inline tslint.json:115]: What's `kebab-case`? And why am I now hungry 😂

File: src/app/communities/communities.component.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<md-card *ngFor="let community of communities">
+  <md-card-header>
+    <img md-card-avatar src="{{ community.logo }}">
+    <md-card-title>
+      {{ community.name }}
+      <md-icon *ngIf="community.codeOfConduct">security</md-icon>
+    </md-card-title>
+    <md-card-subtitle>Beginner, Intermediate</md-card-subtitle>
```
Comment (tanyapowell) [inline src/app/communities/communities.component.html]: I wouldn't hard code this. I'm sure as the list grows there will be some groups that are only for beginners or only for intermediates, etc. I'd grab this from the `communities.json`

Comment (eddiejaoude) [reply src/app/communities/communities.component.html]: OMG! Good spot, I forgot to do that. Thanks 👍

File: tslint.json (hunk 1 above)
Comment (eddiejaoude) [reply tslint.json:115]: Not sure, that was auto generated 🤔

**Review by tanyapowell — APPROVED** [review]

**Pull request merged**